egress/egress-only-tracking.yaml

```diff
@@ -0,0 +1,74 @@
+components:
+  schemas:
+    EgressOnlyTacking:
+      description: |-
+        The list of structures containing Egress offset and it's length to be placed in the packet.
+      type: object
+      required: [port_name]
+      properties:
+        port_name:
+          description: Name of the Rx port where egress only tracking enabled.
+          type: string
+          x-constraint:
+          - '/components/schemas/Port/properties/name'
+          x-field-uid: 1
+        metric_tags:
+          description: |-
+            One or more metric tags can be used to enable tracking portion of or all bits
+            in a corresponding header field for metrics per each applicable value.
+            These would appear as tagged metrics in corresponding egress_only_tracking metrics.
+          type: array
+          items: 
+            $ref: '#/components/schemas/EgressOnlyTacking.MetricTag'
+          x-field-uid: 2
+        filter:
+          description: |-
+            Offset in bits where the tracking value will be placed in the packet.
+          $ref: '#/components/schemas/EgressOnlyTacking.Filter'
+          x-field-uid: 3
+    EgressOnlyTacking.MetricTag:
+      description: |-
+        Metric Tag can be used to enable tracking portion of or all bits
+        in a corresponding header field for metrics per each applicable value.
+        These would appear as tagged metrics in corresponding flow metrics.
+      type: object
+      required:
+      - name
+      properties:
+        name:
+          description: |-
+            Name used to identify the metrics associated with the values applicable
+            for configured offset and length inside corresponding header field
+          type: string
+          pattern: ^[\sa-zA-Z0-9-_()><\[\]]+$
+          x-field-uid: 1
+        offset:
+          description: |-
+            Offset in bits relative to start of corresponding header field
+          type: integer
+          format: uint32
+          default: 0
+          x-field-uid: 2
+        length:
+          description: |-
+            Number of bits to track for metrics starting from configured offset
+            of corresponding header field
+          type: integer
+          format: uint32
+          default: 1
+          minimum: 1
+          x-field-uid: 3
+    EgressOnlyTacking.Filter:
+      type: object
+      properties:
+        choice:
+          description: >-
+            Offset in bits where the tracking value will be placed in the packet.
```

Comment:
Specification of a rule to filter incoming packets on the specified port. If a packet does not match the filter it will not be considered for egress tracking. 
Currently two options are provided: 
none: All packets will be considered for egress only tracking. 
auto.macsec:  This requires that macsec enabled Ethernet interface should be configured on this port. This filter will ensure that only packets with Ethernet Type set to MacSec ( 0xabcd) will be considered for egress only tracking and other packets will be filtered out for the purpose of egress only tracking.

Reply:
Updated.